src/components/SelectorGroup/index.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright 2019, SumUp Ltd.
```

Comment:
Are we changing to 2020? 🙂 
I actually don't know how these licences work... They are renovated/changed annually or what?

Reply:
The copyright year depends on the year the file was first created, so existing files will keep 2019 while new files are generated with 2020.